src/main/java/uk/gov/hmcts/reform/cdapi/config/SwaggerConfiguration.java

```diff
@@ -2,25 +2,45 @@
 
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.web.bind.annotation.RestController;
 import springfox.documentation.builders.PathSelectors;
 import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.service.ApiKey;
+import springfox.documentation.service.SecurityScheme;
 import springfox.documentation.spi.DocumentationType;
 import springfox.documentation.spring.web.plugins.Docket;
 import springfox.documentation.swagger2.annotations.EnableSwagger2;
-import uk.gov.hmcts.reform.cdapi.CommonDataApplication;
+
+import java.util.List;
+import java.util.Optional;
+
+import static com.google.common.collect.Lists.newArrayList;
 
 @Configuration
 @EnableSwagger2
 public class SwaggerConfiguration {
 
     @Bean
     public Docket api() {
+
         return new Docket(DocumentationType.SWAGGER_2)
             .useDefaultResponseMessages(false)
+            .genericModelSubstitutes(Optional.class)
+
             .select()
-            .apis(RequestHandlerSelectors.basePackage(CommonDataApplication.class.getPackage().getName() + ".controllers"))
+            .apis(RequestHandlerSelectors.withClassAnnotation(RestController.class))
             .paths(PathSelectors.any())
-            .build();
+            .build()
+            .securitySchemes(apiKeyList());
+    }
+
+    private  List<SecurityScheme> apiKeyList() {
+        return
+            newArrayList(
+                new ApiKey("Authorization", "Authorization","header"),
+                new ApiKey("ServiceAuthorization", "ServiceAuthorization", "header"),
+                new ApiKey("UserEmail", "UserEmail", "header")
```

Comment:
Any reason for adding this header?

Reply:
Same thing i was about to ask :)

Reply:
I was not aware of it as i copied it from some other project , if not required will remove it now ...Thank you Guys ...